src/api/errors/errors.py

```diff
@@ -0,0 +1,13 @@
+from src.api.errors.error_response import Error
+
+
+class UnauthorizedError(Error):
+    code = 'missing_or_invalid_api_key'
+    message = 'The API key is missing or not recognized'
+    status_code = 401
```

Comment:
I might have missed it but does this actually get used anywhere? Or should it be used in the line in the comments in test_post_air_quality

Reply:
Yes, it's used here: https://github.com/patrickcap/APAQ/pull/34/files#diff-4a8f644b032c2adb9aa1bd80028272b3145771c71962302f44c4d5f9b35e8527R35